cmd/witnessctl/api.go

```diff
@@ -181,7 +181,7 @@ func cfg(ip string, mask string, gw string, dns string) error {
 		return errors.New("Gateway address is invalid")
 	}
 
-	if addr := net.ParseIP(dns); addr == nil {
+	if _, _, err := net.SplitHostPort(dns); err != nil {
```

Comment:
Is it worth returning `fmt.Errorf("DNS... invalid: %v", err)`?

Reply:
I favored consistency with other errors rather than returning a possible irrelevant message (not sure how helpful returned errors are).

Reply:
I guess it's helpful if they eventually become accessible to whoever is trying to figure out what's gone wrong with their network config?
These look like they'll be printed out by `witnessctl`.

Agreed it's worth thinking about whether the err texts from each location will help or confuse, but the ones from `ParseIP` seem like they're not overly cryptic or disconnected (https://cs.opensource.google/go/go/+/refs/tags/go1.20.4:src/net/ipsock.go;l=164)